spec/features/ncr/work_orders/approve_spec.rb

```diff
@@ -21,6 +21,37 @@
     end
   end
 
+  scenario "allows delegate to approve work order" do
+    delegate = create(:user, client_slug: "ncr")
+    approver = work_order.approvers.first
+    approver.add_delegate(delegate)
+    approver.save!
+    login_as(delegate)
+    visit proposal_path(ncr_proposal)
+    click_on('Approve')
+    expect(current_path).to eq(proposal_path(ncr_proposal))
+    expect(page).to have_content("You have approved #{work_order.proposal.public_id}")
+    expect(page).to have_content(delegate.full_name)
+  end
+
+  scenario "delegates can view work order after approval by different delegate" do
+    delegate = create(:user, client_slug: "ncr")
+    delegate_two = create(:user, client_slug: "ncr")
+    approver = work_order.approvers.first
+    approver.add_delegate(delegate)
+    approver.add_delegate(delegate_two)
+    approver.save!
+
+    login_as(delegate)
+    visit proposal_path(ncr_proposal)
+    click_on('Approve')
+    expect(current_path).to eq(proposal_path(ncr_proposal))
+
+    login_as(delegate_two)
+    visit proposal_path(ncr_proposal)
+    expect(page.status_code).to eq(200)
```

Comment:
ok sounds good! in the future I think we are going to try to move away from having status code in feature spec expectations since that isn't a "user-facing feature" but I am cool with keeping this here for now